pkg/runtime/runtime.go

```diff
@@ -939,13 +956,19 @@ func (a *DaprRuntime) publishMessageHTTP(msg *pubsub.NewMessage) error {
 	req.WithHTTPExtension(nethttp.MethodPost, "")
 	req.WithRawData(msg.Data, pubsub.ContentType)
 
-	// TODO Propagate Context
 	ctx := context.Background()
+	spanName := fmt.Sprintf("SubscribedMessage: %s", msg.Topic)
```

Comment:
This is a wrong span name. This is when publishing a message to the app, I think `SubscribeMessage` is either wrong or confusing.

Same for the other instance below.

Reply:
Changed it to PublishMessage. Let me know if this is fine.